### ISSUE TITLE
Image selection issues

### ISSUE THREAD

**Issue by jaweber1** [FrSky-OS/discussion#35]
Couple of things
1 - The list of file names for the image selection is not in the same style as everything else.  I get black letters on a white background
2 - The scroll wheel does not move from top to bottom in the image selection list.  It kind of jumps around to different files.
3 - When I select an image, it is not scaled to fit the screen
4 - After image selection, the OK button should be highlighted, not the modify button which is the state that I'm already in.  Actually, if it's not valid to do CREATE, MODIFY, COPY ETC, then those should not be selectable or just removed from the screen (preferred) until either OK or CANCEL is selected.

**Comment by AloftHobbies** [comment]
Agreed. The navigation in this area should be improved. Very clunky and annoying.

**Comment by mac44mag** [comment]
This may just be me, but one of the reasons that the sim seems a bit clunks stems from the fact that I'm manipulating the joystick/wheel with a mouse.  My sense is that I'll develop the needed touch when I actually have the TX in hand.  

Having said that, the workflow, particularly in this menu, needs a lot of work.
